docs/licensing.asciidoc

```diff
@@ -71,4 +71,19 @@ To avoid any unintended downgrade of individual Elasticsearch clusters to a basi
 we recommend you install the new enterprise license as a new Kubernetes secret next to your existing enterprise license.
 Just replace `eck-license` with a different name in the examples above. ECK will pick the right license automatically.
 
-As soon as you have installed the new license you can safely delete the old license if you want.
\ No newline at end of file
+As soon as you have installed the new license you can safely delete the old license if you want.
+
+[float]
+=== Getting usage data
+The operator periodically writes the total amount of Elastic resources under management to a config map. It is named `elastic-licensing` in the same namespace as the operator. Here is an example of retrieving the data:
+
+[source,shell]
+----
+> kubectl -n elastic-system get configmap elastic-licensing -o json | jq .data
```

Comment:
Would it make sense to avoid an external tool here and just do `-o go-template='{{.data}}`?

Reply:
It doesn't pretty print it though and this is really just for illustrative purposes so people can see the contents. For example
```
09:58 $ kubectl -n elastic-system get configmap elastic-licensing -o go-template='{{.data}}'
map[eck_license_level:basic enterprise_resource_units:1 timestamp:2020-01-07T15:58:20Z total_managed_memory:3.22GB]
```

is a little bit harder to read in the docs IMO, but I see your point

Reply:
No you are right, I did not think about pretty printing